From 2bf84a5524a5a44b0c7da7a2288594598296d161 Mon Sep 17 00:00:00 2001
From: Thor aarch64 build <thor-build@localhost>
Date: Tue, 15 Sep 2026 12:50:20 -0400
Subject: [PATCH 1/8] deps/sqlite: autoreconf -fi before ./configure (aarch64
 libtool 2.4.x split)

The bundled sqlite configure was generated with an older libtool; on Ubuntu 24.04
aarch64 (libtool 2.4.6/2.4.7) it fails to build without regenerating. Run
autoreconf -fi first.
---
 build_files/build_environment/cmake/sqlite.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build_files/build_environment/cmake/sqlite.cmake b/build_files/build_environment/cmake/sqlite.cmake
index 57c3f0fb97f..cc90c55cfa1 100644
--- a/build_files/build_environment/cmake/sqlite.cmake
+++ b/build_files/build_environment/cmake/sqlite.cmake
@@ -62,6 +62,7 @@ ExternalProject_Add(external_sqlite
 
   CONFIGURE_COMMAND ${SQLITE_CONFIGURE_ENV} &&
     cd ${BUILD_DIR}/sqlite/src/external_sqlite/ &&
+    autoreconf -fi &&
     ${CONFIGURE_COMMAND} --prefix=${LIBDIR}/sqlite ${SQLITE_CONFIGURATION_ARGS}
 
   BUILD_COMMAND ${CONFIGURE_ENV} &&
-- 
2.43.0

